CONTRIBUTING.md

```diff
@@ -19,6 +19,74 @@ The Valkey project is led by a Technical Steering Committee, whose responsibilit
 * Want to help with documentation? [Move on to valkey-doc](https://github.com/valkey-io/valkey-doc)
 * Report a vulnerability? See [SECURITY.md](SECURITY.md)
 
+## How to Install clang-format 18 on a Debian Box
```

Comment:
I don't really want to maintain an installation guide of clang-format for debian in this repository. Is there some better place we can point to? Do we need at least clang-format version 18 for this to run correctly?

Reply:
yes 18 is the most compatible one. I think we can go higher but the installation becomes challenging.

What would you recommend for this documentation? Would it work we clarify this being debian specific (so we can extend it in the future to other platforms)?